Assignment/solution_student/css/style.css

```diff
@@ -34,16 +34,145 @@ body {
   color: black;
   font-size: 16px;
   background: white;
+  font-family: 'Open Sans', sans-serif;
+  background-color: #f0efef;
 }
 
+.logo {
+  margin-top: 0;
+  font-size: 24px;
+  font-weight: 800;
+}
+
+/* Buttons */
+.button {
+  height: 40px;
+  width: 180px;
+  color: #033048;
+  font-size: 14px;
+  font-weight: 800;
+  line-height: 40px;
+  border-radius: 5px;
+  text-align: center;
+  background: #f9e42e;
+  text-decoration: none;
+  display: inline-block;
+  transition: color .2s ease-out, background .2s ease-out;
+}
+
+.button:hover {
+  color: #f9e42e;
+  background: #033048;
+}
 /******************************************
 /* LAYOUT
 /*******************************************/
+.inner-container {
+  margin: 0 auto;
+  padding: 0 116px;
+  max-width: 1200px;
+}
+
+.floatleft {
+	float: left;
+}
 
+.floatright {
+	float: right;
+}
 /******************************************
 /* PAGES
 /*******************************************/
 
 /******************************************
 /* SECTIONS
-/*******************************************/
\ No newline at end of file
+/*******************************************/
+header {
+  height: 80px;
+  background-color: #033048;
+  color: white;
+  font-weight: bolder;
+}
+
+header h1 {
+	font-size: 30px;
+	float: left;
+	padding: 0px 10px;
+	font-weight: bolder;
+}
+
+header ul {
+	 float: right;
+}
+
+header li {
+	font-size: 24px;
+	margin: 0 10px;
+	padding: 8px 0;
+	color: #f9e42e;
+	list-style: none;
+ 	display: inline-block;
+}
+
+
+
+article {
+	width: 700px;
+	margin: 20px;
+	color: #606161;
+	margin-bottom: 80px;
+}
+
+article h2 {
+	color: #121212;
+	font-size: 40px;
+	font-weight: normal;
+}
+
+aside {
+	width: 240px;
+	margin: 20px;
+}
+
+aside li {
+	list-style: none;
+}
+
+.signup {
+  padding: 20px 0;
+  text-align: center;
+  background: #f9e42e;
+  position: relative;
+  top: -40px;
```

Comment:
Okay, so the reason your .signup div is ending up spaced away from the top of the footer is because on your footer you have a `padding: 40px 0;` which is _40px top_, 40px bottom, and 0  on left and right. So, just change that to `padding: 0 0 40px;` which is '0 padding top, 0 padding left/right, 40px bottom'. Then you'll need to add some space between this signup section and the rest of the footer, so you'll need to add some `margin-bottom` to `.signup`